src/jit/morph.cpp

```diff
@@ -6953,6 +6958,10 @@ GenTreePtr          Compiler::fgMorphCall(GenTreeCall* call)
         {
             szFailReason = "Localloc used";
         }
+        else if ((call->gtFlags & GTF_CALL_NO_RETURN) != 0)
+        {
+            szFailReason = "No return";
+        }
```

Comment:
Can you explain what motivated this change? Is there some downside to tail calling a no-return method (eg larger calling sequence?)